blocks/form/form.css

```diff
@@ -8,13 +8,13 @@ main .form > div {
   margin-left: 35%;
 }
 
-main .form h2 {
+main .form h3 {
```

Comment:
Is this necessary due to an authoring decision or has anything been update from a code perspective?

Reply:
The authored document now uses `h3` instead of `h2`.